sefaria/model/text.py

```diff
@@ -5675,22 +5681,45 @@ def get_simple_term_mapping_json(self, rebuild=False):
         return self._simple_term_mapping_json
 
     def get_term(self, term_name):
+        """
+        Returns the full term, if mapping not present, builds the full term mapping.
+        :param term_name: String
+        :returns: full Term (Mongo Record)
+        """
         if not self._full_term_mapping:
             self.build_term_mappings()
         return self._full_term_mapping.get(term_name) if term_name in self._full_term_mapping else Term().load({"name": term_name})
 
 
 
     def get_topic(self, slug):
+        """
+        Returns a dictionary containing with keys "en" and "he".
+        The "en" field has a value of the topic's English primary title, and the "he" field has a
+        value of the topic's Hebrew primary title.
+        :param slug: String
+        :returns: topic map for the given slug Dictionary
+        """
         return self._topic_mapping[slug]
 
     def get_topic_mapping(self, rebuild=False):
+        """
+        Returns the topic mapping if it exists, if not rebuilds it and returns
+        :param rebuild: Boolean (optional, default set to False)
+        """
         tm = self._topic_mapping
         if not tm or rebuild:
             tm = self._build_topic_mapping()
         return tm
 
     def _build_topic_mapping(self):
+        """
+        Builds the topic mapping. The topic mapping is a dictionary with one key, the slug of the topic.
```

Comment:
The dictionary has many keys, one for each topic.

Reply:
Ah, I misunderstood - thank you for pointing that out!